pkg/ddevapp/config_test.go

```diff
@@ -1317,3 +1319,220 @@ func TestDatabaseConfigUpgrade(t *testing.T) {
 		assert.Empty(app.MariaDBVersion)
 	}
 }
+
+// TestConfigMergeStringList verifies that string merges update w/o clobber
+func TestConfigMergeStringList(t *testing.T) {
+	assert := asrt.New(t)
+	pwd, _ := os.Getwd()
+
+	projDir, err := filepath.Abs(testcommon.CreateTmpDir(t.Name()))
+	require.NoError(t, err)
+
+	err = fileutil.CopyDir("./testdata/TestConfigMergeItems/.ddev", filepath.Join(projDir, ".ddev"))
```

Comment:
Please use separate testdata for separate tests, even if they're identical. That way fiddling with the testdata for one test won't break another one. And use t.Name() here instead of the literal `TestConfig...`, so if you happen to change the test name it can follow you.

Reply:
Done.